src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/SpanSelectorLabel.tsx

```diff
@@ -0,0 +1,32 @@
+import { css } from '@emotion/css';
+import { Button, Tooltip, useStyles2 } from '@grafana/ui';
+import { useSpanSelectorFromUrl } from '@shared/domain/url-params/useSpanSelectorFromUrl';
+import React from 'react';
+
+export function SpanSelectorLabel() {
+  const styles = useStyles2(getStyles);
+  const [spanSelector, setSpanSelector] = useSpanSelectorFromUrl();
+
+  return (
+    <div className={styles.container}>
+      <Tooltip content={`You have added a span selector to the flamegraph query (${spanSelector}).`} placement="top">
```

Comment:
It's not very clear in the UI that span selector was applied 🤔 Also using the example it didn't find the data for that span. We could be more clear about it as well. It's weird also that the link "profiles for this span" renders different result than "Open in Profiles Drilldown"

<img width="1698" alt="Screenshot 2025-03-03 at 15 45 10" src="https://github.com/user-attachments/assets/cbc1bb48-b39f-49a0-b047-2c972a721ac3" />